pyserini/vectorizer.py

```diff
@@ -103,4 +118,53 @@ def get_vectors(self, doc_ids: List[str]):
                 matrix_data.append(tfidf)
 
         vectors = csr_matrix((matrix_data, (matrix_row, matrix_col)), shape=(num_docs, self.vocabulary_size))
-        return self.l2norm(vectors)
+        return self._l2normalize(vectors)
+
+
+class BM25Vectorizer(VectorizerBase):
+    """Wrapper class for bm25 vectorizer implemented on top of Pyserini.
+
+    Parameters
+    ----------
+    lucene_index_path : str
+        Path to lucene index folder
+    min_df : int
+        Minimum acceptable document frequency
+    verbose : bool
+        Whether to print out debugging information
+    """
+
+    def __init__(self, lucene_index_path: str, min_df: int = 1, verbose: bool = False) -> None:
+        super().__init__(lucene_index_path, verbose, min_df)
+
+    def get_vectors(self, doc_ids: List[str]):
+        """Get the bm25 vectors given a list of doc_ids
+
+        Parameters
+        ----------
+        doc_ids : List[str]
+            The piece of text to analyze.
```

Comment:
isn't this the list of docids?
btw, can you change `doc_ids` to `docids`? I use the latter consistently.